packages/perseus/src/README.md

```diff
@@ -1,7 +1,39 @@
-Widgets from the Perseus (exercise) editor.
```

Comment:
This sounds like the top-level README for the entire repo. As such, let's move it right up to the root of the repo.  That should also fix up the image references as they'll then be in the same place.